cli/config.toml

```diff
@@ -0,0 +1,30 @@
+title = "CLI config file"
+
+#Flags
+[[flags]]
+token = ""
+[remotes]  # root flags
+things_url = "http://localhost"
+users_url = "http://localhost"
+reader_url = "http://localhost"
+http_adapter_url = "http://localhost/http"
+bootstrap_url = "http://localhost"
+certs_url = "http://localhost"
+
+[filter]  # Client
+offset = ""
+limit = ""
+raw_output = ""
+name = ""
+contact = ""
+email = ""
+metadata = ""
+msg_content_type= ""
+tls_verification = "false"
+
+[channel]  # Channel flags
+status = ""
+state = ""
+topic = ""
+
```

Comment:
After chaning the config  via command,  order in which configured stored are changed to alphabetically 

Example : After applying the following command`./build/mainflux-cli  config  things_url http://localhost:8999/things` you may notice that order of all the configuration has been changed

So please change the configuration order in which cobra generates

Reply:
Okay